tools/dependencies/known-dependencies.txt

```diff
@@ -707,4 +707,17 @@ zookeeper-3.4.6.jar
 zookeeper-3.5.9.jar
 zookeeper-jute-3.5.9.jar
 zstd-jni-1.3.3-1.jar
-zstd-jni-1.4.3-1.jar
\ No newline at end of file
+zstd-jni-1.4.3-1.jar
```

Comment:
The corresponding dependencies should be put into the corresponding lines, according to lexicographical order